terraform/gcp/projects/basehub-template.tfvars

```diff
@@ -0,0 +1,31 @@
+prefix                 = "{{ cluster_name }}"
+project_id             = "{{ cluster_name }}"
+
+zone                   = "{{ cluster_region }}-b"
```

Comment:
We should probably just explicitly ask for the zone as well - in us-central1-b, we can't have GPUs :(